### ISSUE TITLE
[OPS] Email tag {$authorOrcidUrl} returns invalid URL when used as a link

### ISSUE THREAD

**Issue by alexxxmendonca** [pkp/orcidProfile#122]
**Describe the bug**
The tag `{$authorOrcidUrl}` generated a dynamic link that enables authors to connect their ORCID iD via the API. In order to make a more user-friendly link, instead of using the tag itself, the template suggests to add it to a link (ie: within an `a href`).

However, the platform is automatically adding a `../management/settings/` before the `{$authorOrcidUrl}` tag, which invalidates the URL.

**To Reproduce**
1. Check your ORCID email templates
2. Check the link to connect ORCID ID (usually the one followed by the ORCID icon)
3. See how it points to  `../management/settings/{$authorOrcidUrl}`, which is not a valid URL

**What application are you using?**
Open Preprint Systems 3.2.0.1

**Additional information**
![Workflow Settings](https://user-images.githubusercontent.com/4059679/79562894-24386180-8082-11ea-95c5-fc1ea1f51f8e.gif)

**Comment by librariam** [comment]
+1 from PKP|PS, same issue is reported by a journal on OJS 3.2.0.2 with ORCID plugin v1.1.2.3

Also related reports [in this Forum thread](url)

**Comment by withanage** [comment]
Hi @librariam @alexxxmendonca ,

I have tested the OPS for this problem yesterday.

I have looked into the problem, as you have described and it i  a general  problem in the template mechanism that always the current url gets added tp any  tiny mce links (both in OPS and OJS  3.2  +)  

Problem is not only orcid  specific , other emails we did not have the links, therefore it was not visible.

I  will talk with Nate about this and come back to you soon.
